static/app/views/releases/utils/index.tsx

```diff
@@ -257,3 +257,34 @@ export function isVersionInfoSemver(
 ): versionInfo is SemverVerison {
   return versionInfo.hasOwnProperty('components');
 }
+
+/**
+ * Translates seconds into human readable format of seconds, minutes, hours, days, and years
+ *
+ * @param  {number} seconds The number of seconds to be processed
+ * @return {string}         The phrase describing the amount of time
+ */
+export function forHumans(seconds) {
```

Comment:
We already have this no? https://github.com/getsentry/sentry/blob/9c150d4c40a00aaf362b4c57b3a18edb05903f72/static/app/utils/formatters.tsx#L67-L80

Reply:
Oh huh.
looks like i'm looking for `getExactDuration`

Maybe i'll separate this out into a separate PR.
the formatter i copied seems to do this in a more elegant way than our current formatter

Reply:
Eh nah i'll just use getExactDuration